pkg/controller/storagecluster/controller_test.go

```diff
@@ -1120,28 +1120,28 @@ func TestStorageClusterDefaultsMaxStorageNodesPerZone(t *testing.T) {
 	err = controller.setStorageClusterDefaults(cluster)
 	require.NoError(t, err)
 	require.NotEqual(t, (*corev1.CloudStorageSpec)(nil), cluster.Spec.CloudStorage)
-	require.Equal(t, uint32(3), *cluster.Spec.CloudStorage.MaxStorageNodesPerZone)
+	require.Equal(t, uint32(6), *cluster.Spec.CloudStorage.MaxStorageNodesPerZone)
 
 	err = controller.setStorageClusterDefaults(cluster)
 	require.NoError(t, err)
 	require.NotEqual(t, (*corev1.CloudStorageSpec)(nil), cluster.Spec.CloudStorage)
-	require.Equal(t, uint32(3), *cluster.Spec.CloudStorage.MaxStorageNodesPerZone)
+	require.Equal(t, uint32(6), *cluster.Spec.CloudStorage.MaxStorageNodesPerZone)
```

Comment:
Are these lines duplicated on purpose or just a copy paste error?

Reply:
I think there was some logic before which made them different. Currently it's just a duplicate. Will remove this.